colcon_ros_cargo/task/ament_cargo/build.py

```diff
@@ -104,8 +105,17 @@ def write_cargo_config_toml(package_paths):
     config_dir = Path.cwd() / '.cargo'
     config_dir.mkdir(exist_ok=True)
     cargo_config_toml_out = config_dir / 'config.toml'
-    cargo_config_toml_out.unlink(missing_ok=True)
-    toml.dump(content, cargo_config_toml_out.open('w'))
+    # missing_ok flag was introduced in Python 3.8, older versions
+    # raise a FileNotFoundError exception
+    if sys.version_info.minor >= 8:
+        cargo_config_toml_out.unlink(missing_ok=True)
```

Comment:
So we delete the previous `config.toml`if it exists?

Nit: Perhaps unnecessary for folks who spend a lot of time in python, but I would suggest wrapping this in a small free function with a clear name.

`def _delete_file(f)` or something similar.

Also, we should probably explain why we pass on the exception 
`# No work for us to do, the file already doesn't exist` or something like that.

Reply:
After looking at this further I feel the step might not be necessary, unless there is some extreme corner case, opening a file as `w` by default already deletes its whole content and replaces it, so I removed the whole block in https://github.com/colcon/colcon-ros-cargo/pull/22/commits/0612fda1daa42202bdc2b89e30c75b1f2e11bdbf

The only case in which this would behave differently (I believe) is if users created a `.cargo/config.toml` that was a symlink to their own file, then ran a `colcon build`, previously the link would be deleted and a new config file would be created, now the symlink would be followed and edited.
I'd argue that it is such a corner case that I don't even know what the intended behavior would be